.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,88 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes and platform settings. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+{Screenshot of audit logs page}
+
+---
+
+## Logged events
+
+(Should we include the technical names of the events? Or maybe one example event per type? i.e. `datasource.created` etc.)
+
+There are several types of events tracked in Appsmith's audit logs:
+
+- App-specific activity
+    - Changes to pages/widgets (`page.updated`)
+    - Datasource configuration(`datasource.updated`)
+    - Running queries(`query.executed`)
+    - Deleting apps
+
+- Admin settings changes
+    - Who changed certain settings (`instance_setting.updated`)
+    - Changes/new groups or roles (`group.updated`)
+
+## Event details
+
+Each logged item has a number of data points that describe that event, as well as any related apps, datasources, queries, and workspaces.
+
+| Data | Description | Details |
+|------|-------------|---------|
+| event | Each action performed on Appsmith app(edit or view mode) is classified as event |  |
+| timestamp | Displays the date and time of a logged event. |  |
+| user | Displays the user who made performed this event | `id`, `email`, `name`, `ipAddress` |
+| resource | The type of resource on which this event was performed (Query, Datasource, etc.) | `id`, `type`, `name` |
+| app | Application on which the action is performed. | `id`, `name`, `git` {`branch`, `default`} |
+| workspace | Workspace on which the action is performed. | `id`, `name` |
+| metadata | Platform details, such as the Appsmith version. | `ipAddress`, `appsmithVersion`, `createdAt` |
+
+Below is an example of a log item:
+
+```JSON
+{
+    "event": "datasource.created",
+    "timestamp": "2022-06-29T08:36:33.507+00:00",
+    "user": {
+        "id": "62bc0f11545c4c00b19d5c65",
+        "email": "john@appsmith.com",
+        "name": "John Doe",
+        "ipAddress": "223.183.39.7"
+    },
+    "resource": {
+        "id": "62bc0f11545c4c00b19d5c59",
+        "type": "Datasource",
+        "name": "Movies"
+    },
+    "app": {
+        "id": "62bc0f11545c4c00b19d5c68",
+        "name": "Standup App",
+        "git": {
+            "branch": "feat/new-ui",
+            "default": "master"
+        }
+    },
+    "workspace": {
+        "id": "62bc0f11545c4c00b19d5c64",
```

Comment:
@spark-c Can we remove all `ipAddress` references since that is not part of v1. Also from the JSON payload as well.

Reply:
@spark-c also this looks like it's been taken from the PRD and is not the final output. 

Here's an actual output from our deploy preview `datasource.created` event looks like, please edit and use this

```
{
  "event": "datasource.created",
  "timestamp": "13/10/2022, 15:34:10",
  "resource": {
    "id": "6347e29ade6bb67831ca58b1",
    "type": "Datasource",
    "name": "Offers",
    "datasourceType": "MongoDB"
  },
  "workspace": {
    "id": "6347a101de6bb6523a1v5779",
    "name": "Backoffice"
  },
  "user": {
    "id": "6347a101de6bb231da5777",
    "email": "jane.doe@example.com"
  },
  "metadata": {
    "appsmithVersion": "v1.8.4"
  },
  "id": "6347e29ade6bb6534rca58b2"
}
```